go/noise/gaussian_noise.go

```diff
@@ -155,6 +181,23 @@ func addGaussian(x, sigma float64) float64 {
 	return roundToMultipleOfPowerOfTwo(x, granularity) + float64(sample)*granularity
 }
 
+// computeConfidenceIntervalGaussian computes a confidence interval that contains the raw value x from which
+// float64 noisedX is computed with a probability equal to 1 - alpha with the given sigma.
+func computeConfidenceIntervalGaussian(noisedX, sigma, alpha float64) ConfidenceInterval {
+	z := inverseCDFGaussian(sigma, alpha/2)
+	FloatLowerBound := noisedX + z
+	FloatUpperBound := noisedX - z
+	FloatInterval := ConfidenceInterval{LowerBound: FloatLowerBound,
+		UpperBound: FloatUpperBound}
+	return FloatInterval
+}
+
+// inverseCDFGaussian computes the quantile z satisfying Pr[Y <= z] = p for a random variable Y that is Gaussian
+// distributed with the specified sigma.
+func inverseCDFGaussian(sigma, p float64) float64 {
+	return -sigma * math.Sqrt(2) * math.Erfcinv(2.00*p)
```

Comment:
Does this need to be `2.00`? Can we not use `2`?

Reply:
It should be just fine with 2 as well :)